src/backend/joanie/tests/core/test_api_admin_products.py

```diff
@@ -148,3 +148,21 @@ def test_admin_api_product_delete(self):
         response = self.client.delete(f"/api/v1.0/admin/products/{product.id}/")
 
         self.assertEqual(response.status_code, 204)
+
+    def test_admin_api_product_bulk_delete(self):
+        """
+        Staff user should be able to delete multiple products.
+        """
+        admin = factories.UserFactory(is_staff=True, is_superuser=True)
+        self.client.login(username=admin.username, password="password")
+        products = factories.ProductFactory.create_batch(3)
+        factories.ProductFactory()
+        data = {"id": [str(product.id) for product in products]}
+        response = self.client.delete(
+            "/api/v1.0/admin/products/", data=data, content_type="application/json"
+        )
+        content = response.json()
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(models.Product.objects.count(), 1)
+        self.assertEqual(len(content["deleted"]), 3)
+        self.assertFalse("error" in content)
```

Comment:
Maybe we should add a test method about `test_admin_api_course_runs_bulk_delete_error` to this class `ProductAdminApiTest` ?

Reply:
This comment will be resolved once MoneyFields have been deleted